earth_enterprise/src/fusion/autoingest/sysman/AssetHandleD.h

```diff
@@ -46,7 +46,10 @@ class DerivedAssetHandleD_ : public virtual BaseD_, public ROBase_
   typedef typename BBase::HandleType HandleType;
  public:
   virtual HandleType Load(const std::string &boundref) const {
-    return HandleType(Impl::Load(boundref));
```

Comment:
Is this load function still necessary? It seems that it could be removed.

Reply:
Turns out it wasn't necessary. I took that out and the base class version in AssetHandle.h.